### PR TITLE
upipe_sync: module to do A/V sync

### PR DESCRIPTION
**Pull request by funman**

### CODE REVIEW

File: lib/upipe-modules/upipe_sync.c [hunk 1]
```diff
@@ -0,0 +1,1083 @@
...
+    uint64_t latency;
+    if (!ubase_check(uref_clock_get_latency(flow_def, &latency)))
+        latency = 0;
+
+    flow_def = uref_dup(flow_def);
+    if (!flow_def)
+        return UBASE_ERR_ALLOC;
+
+    // FIXME : estimated latency added by processing
+    latency += UCLOCK_FREQ / 25;
```
Comment (cmassiot) [inline lib/upipe-modules/upipe_sync.c]: Shouldn't this be based on fps ?

Comment (funman) [reply lib/upipe-modules/upipe_sync.c]: Yes

Comment (cmassiot) [reply lib/upipe-modules/upipe_sync.c]: Do you intend to fix it ?

Comment (funman) [reply lib/upipe-modules/upipe_sync.c]: Of course but will need some time to test it properly

Comment (cmassiot) [reply lib/upipe-modules/upipe_sync.c]: Isn't it just a matter of adding * fps.den / fps.num ?

Comment (funman) [reply lib/upipe-modules/upipe_sync.c]: Yes but I remember having problems with this so want to test for a while

Comment (cmassiot) [reply lib/upipe-modules/upipe_sync.c]: Do you want to push it as is ?

Comment (funman) [reply lib/upipe-modules/upipe_sync.c]: Yes I guess it doesn't harm

Comment (cmassiot) [reply lib/upipe-modules/upipe_sync.c]: Can you squash it ?

**Pull request merged**